### DISCUSSION TITLE
EMQX MQTT Cluster Setup with Home Assistant EMQX Add-On

### DISCUSSION THREAD

**Discussion by hanley-development** [emqx/emqx#11707] [Q&A]
Wondering if anyone is has gotten EMQX clustering to work with the addon and another docker container?

I keep getting erconnectrefuse errors and wondering if it is because of env_vars.

```
- name: EMQX_NODE__NAME
  value: emqx2@192.168.0.140

- name: EMQX_NODE__COOKIE
  value: <changed>

- name: EMQX_CLUSTER__DISCOVERY_STRATEGY
  value: static

- name: EMQX_CLUSTER__STATIC__SEEDS
  value: "["emqx@192.168.20.86", "emqx2@192.168.0.140"]"
```

> 2023-09-28T12:30:51.268133-05:00 [error] crasher: initial call: gen_rpc_client:init/1, pid: <0.17899.0>, registered_name: [], exit: {{badrpc,econnrefused},[{gen_server,init_it,6,[{file,"gen_server.erl"},{line,407}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}, ancestors: [gen_rpc_client_sup,gen_rpc_sup,<0.1883.0>], message_queue_len: 0, messages: [], links: [<0.1889.0>], dictionary: [], trap_exit: true, status: running, heap_size: 2586, stack_size: 29, reductions: 7419; neighbours:

**Comment by sstrigler** [comment]
Hi, 

it looks like your two nodes are on different networks. Are they both docker containers? Is there basic network connectivity between both instances?

Also could you please clarify what you mean with "the addon"?

**Comment by sstrigler** [comment]
Seems to be a duplicate and you got an answer at the reported issue already: https://github.com/emqx/emqx/issues/11708
